tests/conftest.py

```diff
@@ -1,5 +1,5 @@
 # This is a special pytest file
```

Comment:
is this a module comment? It should probably be a module _docstring_, and it's also kind of saying nothing at all. What does this special file do?